File: 이유진/codes/Lv.1/lotto.java
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        
        int zer_num = 0; // 0의 개수
        int ans = 0; // 일치하는 것의 개수
        for(int i=0; i<6; i++){
            if(lottos[i] == 0) zer_num ++;
            for(int j = 0; j<6; j++){
                if(win_nums[j] == lottos[i]){
                    ans++;
                    break;
                }else continue;
            }
        }
        
        int max = zer_num + ans;
        int min = ans;
        int max_result, min_result;
        
        if((7-max)>=6){
            max_result = 6;
        }else max_result = 7-max;
        
        if((7-min)>=6){
            min_result = 6;
        }else min_result = 7-min;
         
        int[] answer = new int[2];
        answer[0] = max_result;
        answer[1] = min_result;
        return answer;
    }
}

File: 김정희/Lv.1/로또/lotto.py
def check(n):
    if n == 6:
        return 1
    elif n == 5:
        return 2
    elif n == 4:
        return 3
    elif n == 3:
        return 4
    elif n == 2:
        return 5
    else:
        return 6


def solution(lottos, win_nums):
    remain = 6
    for k in lottos:
        if k in win_nums:
            remain -= 1
    correct = 6 - remain

    zero = lottos.count(0)

    if zero >= remain:
        best = correct + remain
    else:
        best = zero + correct

    answer = [check(best), check(correct)]
    return answer


print(solution([1, 2, 0, 0, 5, 6], [5, 6, 7, 8, 9, 10]))